### DISCUSSION TITLE
offline SO. Admin screen black

### DISCUSSION THREAD

**Discussion by RadarG1** [Security-Onion-Solutions/securityonion#2467] [General] [answered]
I just installed an offline SO VM (vmware) using the ISO. The admin web page is all black any ideas how to fix this?

**Comment by dougburks** [comment, marked as answer]
Please provide more detail.

Are you able to see the login page at all?  If so, are you able to login?

What web browser are you using? Any errors in the browser console?

Have you tried a different browser?

**Reply by RadarG1** [reply]
Edge and IE. I later tried firefox and had better results. thank you for your time
